Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -173,6 +173,15 @@ define([
             get : function() {
                 return this._readyPromise.promise;
             }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
```

Comment:
OK, so they are wrong too, we'll fix that when 3D-tiles get PRed into master but no need to add to it.